.github/ISSUE_TEMPLATE/question.yaml

```diff
@@ -0,0 +1,14 @@
+name: Ask a question
+description: Please use the community forum (https://community.cesium.com/) for general questions about using Cesium.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        :exclamation: Please use the [community forum](https://community.cesium.com/) for asking questions about how to use Cesium and best practices. The core Cesium team actively monitors the forum and we love seeing what people are working on! :exclamation:
+  - type: checkboxes
+    id: verify
+    attributes:
+      label: Verification
+      description: Confirm you will ask your question on our forums linked above
+      options:
```

Comment:
No, I really tried that but forms must have at least one "input" type section. I went with the checkbox as it's the most minimal one allowing no custom text or anything. I tried to phrase it such that it further encourages users not to actually submit.
This one was a little tricky so I'm definitely open to ideas.
We could just drop this type completely if we want, not sure if that's better or worse for discoverability of the forums. We could also easily add a link in the other templates but I'm not sure if that'll get read or just glossed over

Reply:
I figured, but just wanted to confirm. If that's the case, this is fine as-is.

I do think it's worth having the option, so that people click it and then understand that the forum exists.